src/components/pages/rules/flyouts/ruleEditor/actionEmailSetup.js

```diff
@@ -40,10 +52,14 @@ export class ActionEmailSetup extends Component {
     const showSetupIncomplete = !showPending && !showError && !actionsEmailIsEnabled && actionPollingTimeout;
     const showSetupComplete = !showPending && !showError && actionsEmailIsEnabled;
 
+    // Only show setup information if the application was deployed as owner,
+    // as if it was not cannot verify whether or not setup is complete.
+    // Also only show verification of setup if setup button was clicked.
     return (
-      <div className="action-email-setup-container">
+      isDeployedAsOwner
+      ? <div className="action-email-setup-container">
         {
-          showPending &&
+          showPending && clickedSetup &&
```

Comment:
Couldn't you add clickedSetup to the calculation for showPending?

Reply:
good call